ccpp/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@ if (FV3)
         driver/GFS_init.F90
         CCPP_driver.F90
         )
+
```

Comment:
@DusanJovic-NOAA I had to add this line back in. 
Without this, the fast-physics called from FV3 will try to open a new parallel region in [fv_sat_adj.F90](https://github.com/ufs-community/ccpp-physics/blob/ufs/dev/physics/MP/GFDL/fv_sat_adj.F90#L324), which causes failure for the suites using GFDL MP w/ fast-physics, mostly HAFS configurations.